### ISSUE TITLE
Prod build includes local template paths

### ISSUE THREAD

**Issue by Manduro** [driftyco/ionic-app-scripts#35]
After running `ionic-app-scripts build`:
```
ionic-app-scripts 0.0.23
build prod started ...
```
My `build/main.js` has a lot of these in there:
```ts
t.createRenderComponentType("/Users/etc/etc/etc/etc/etc/node_modules/ionic-angular/components/alert/alert-component.d.ts class AlertCmp - inline template",0,vb.None,Lyt,{})
```

This same thing is mentioned here: https://github.com/angular/angular/issues/11117, but was seemingly fixed.

~~Oh and the prod build also seems to generate a `main.js.map`, which doesn't seem necessary. It makes me wonder if a prod build might be a dev build after all.~~ edit: PR for this in #40

**Comment by mlynch** [comment]
This might just be a case of us using a slightly older angular sub dependency. Thoughts @danbucholtz @adamdbradley?

**Comment by benelliott** [comment]
The thread you linked to includes the commit that fixes this issue, which was merged on Sept 28. There hasn't been an "official" release of Angular since Sept 23. Are you using the nightly builds? If not you won't see this change take effect until the next release.

**Comment by Manduro** [comment]
@benelliott Ah, missed the dates there. Thanks!

**Comment by danbucholtz** [comment]
@benelliott, good catch.  @Manduro, I'm going to go ahead and close for now. Let's revisit if it's an issue after the next Angular release.  Sound good?  Otherwise, say the word and I'll keep it open.

Thanks,
Dan

**Comment by Manduro** [comment]
This is fixed in Angular 2.0.2 :)
